client/js/components/show-hide.js

```diff
@@ -11,8 +11,24 @@ const showHide = (state) => {
   const init = () => {
     el.setAttribute('aria-haspopup', 'true');
     options.trigger.setAttribute('aria-expanded', getActive());
+
+    setDataTrackAria(getActive());
   };
 
+  function setDataTrackAria(value) {
+    const dataTrackOptions = options.trigger.getAttribute('data-track-click');
+
+    if (!dataTrackOptions) return;
+
+    const updatedTrackOptions = JSON.stringify(Object.assign(
+      {},
+      JSON.parse(dataTrackOptions),
+      {actionFromAriaExpanded: value ? 'open' : 'close'}
```

Comment:
Agree – this is why I changed to `actionFromAriaExpanded: 'open'` and `close` from `ariaExpanded: true` (I felt that having 'action' in the label made it clearer). I'm not convinced `isOpen` is any less ambiguous. I'd say your first suggestion, `wasOpened` is better. Maybe `didOpen` is best?